pkgs/development/python-modules/nimbo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, python3, awscli }:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
```suggestion
{ lib, buildPythonApplication, pythonOlder, fetchFromGitHub, setuptools, boto3, awscli, requests, click, pyyaml, pydantic, awscli }:

buildPythonApplication rec {
```

Reply:
This is a python application and is called from `all-packages.nix`, I don't know if I can use this style of arguments unless it's called from `python-packages.nix`?

Reply:
IIRC, there's a preference to bring in packages from the related package scope. In this case `pkgs.callPackage`. So the existing inputs make sense.

Reply:
I stand corrected, this is in `python-modules`, yes, please import the individual python packages.

Reply:
> This is a python application and is called from `all-packages.nix`, I don't know if I can use this style of arguments unless it's called from `python-packages.nix`?

If you want this as a module and an application, do you mind adding the following to `pkgs/top-level/all-packages.nix`:
```
  <pkg-name> = with python3Packages; toPythonApplication <python-module>;
```
More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

If you want it just as an application, then it shouldn't go into `python-modules`, but a related `pkgs/` directory.